params/config.go

```diff
@@ -90,7 +90,7 @@ var (
 		EIP155Block:         big.NewInt(3),
 		EIP158Block:         big.NewInt(3),
 		ByzantiumBlock:      big.NewInt(1035301),
-		ConstantinopleBlock: nil,
+		ConstantinopleBlock: big.NewInt(3660663),
```

Comment:
@karalabe I'm so curious to know the reason why we want to use palindrome number for rinkeby testnet? 😄 thanks!

Reply:
It's fun :)

Reply:
haha, I also thought it's just for fun 👍

Reply:
Hello, I would like to have my ethereum node, I am from Latin America, how can you help me configure correctly?
I want to contribute to the network with a node.
regards...